### PR TITLE
Push new tag of docker image when releasing nebula version

### PR DESCRIPTION
**Pull request by yixinglu**
- Add new `release` event listener for docker image building jobs
- Fix docker context in building script
- Add docker image address in readme

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ name: docker
 on:
   schedule:
     - cron: '0 18 * * *'
+  release:
```
Comment (Shylock-Hg) [inline .github/workflows/docker.yaml]: When will be triggered?

Comment (yixinglu) [reply .github/workflows/docker.yaml]: next time when release will be created.

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -24,10 +27,20 @@ jobs:
...
         run: |
           docker build -t ${IMAGE_NAME} -f docker/Dockerfile.${{ matrix.service }} .
           docker push ${IMAGE_NAME}
         shell: bash
+      - name: build release image
+        if: github.event_name == 'release'
+        env:
+          IMAGE_NAME: ${{ secrets.DOCKER_USERNAME }}/nebula-${{ matrix.service }}
+        run: |
+          TAG=$(git describe --exact-match --abbrev=0 --tags)
```
Comment (Shylock-Hg) [inline .github/workflows/docker.yaml]: Will the `TAG` empty?

Comment (yixinglu) [reply .github/workflows/docker.yaml]: No. Because this job will be triggered by release event which will create a new tag on the current commit.

Comment (yixinglu) [reply .github/workflows/docker.yaml]: No. Because this job will be triggered by release event which will create a new tag on the current commit.

**Pull request merged**